### PR TITLE
Feature/lens

### PR DESCRIPTION
**Pull request by skuhlmann**
## GitHub Issue

[GitHub issue](https://github.com/HausDAO/daohaus-monorepo/issues/626).

## Changes

- swaps ceramic fetch for lens
- new lens graphql types and query
- propagates chanegs

## Packages Added

- none
## Checks

Before making your PR, please check the following:

- [x] Critical lint errors are resolved
- [x] App runs locally
- [ ] App builds locally (run the build command for *any impacted package* and check for any errors before the PR)

### CODE REVIEW

File: libs/dao-data/src/subgraph/queries-lens/profiles.generated.ts [hunk 1]
```diff
@@ -0,0 +1,345 @@
...
+      id: any;
+      name?: string | undefined;
+      bio?: string | undefined;
+      followNftAddress?: any | undefined;
+      metadata?: any | undefined;
+      isDefault: boolean;
+      handle: any;
+      ownedBy: any;
+      picture?:
+        | {
```
Comment (brossetti1) [inline libs/dao-data/src/subgraph/queries-lens/profiles.generated.ts:21]: never seen this syntax before, is the initial `|` needed here?

File: libs/dao-data/src/utils/transformers.ts [hunk 2]
```diff
@@ -25,18 +25,21 @@ export const transformProposal = (
...
-  ens: string | null,
-  profile: BasicProfile
+  profile: LensProfile
 ): AccountProfile => {
   return {
     address,
-    ens,
-    ...profile,
+    name: profile?.name,
+    ens: profile?.onChainIdentity.ens?.name,
```
Comment (brossetti1) [inline libs/dao-data/src/utils/transformers.ts:33]: might want to add `?.ens` here in case onChainIdentity is empty

**Review by brossetti1 — APPROVED** [review]
left a few comments to check out, approving regardless

File: libs/dao-data/src/subgraph/queries-lens/profiles.generated.ts (hunk 1 above)
Comment (skuhlmann) [reply libs/dao-data/src/subgraph/queries-lens/profiles.generated.ts:21]: it's generated so i can't really control that.

**Pull request merged**